template_feed/Microsoft.DotNet.Test.ProjectTemplates.8.0/Microsoft.DotNet.Test.ProjectTemplates.8.0.csproj

```
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>netstandard1.0</TargetFramework>
+        <IncludeBuildOutput>False</IncludeBuildOutput>
+        <IncludeSource>False</IncludeSource>
+        <GenerateAssemblyInfo>False</GenerateAssemblyInfo>
+        <OutputPath>$(TemplatesFolder)</OutputPath>
+        <EnableDefaultItems>False</EnableDefaultItems>
+        <IsPackable>true</IsPackable>
+        <NoPackageAnalysis>true</NoPackageAnalysis>
+        <NoWarn>2008;NU5125</NoWarn>
+
+        <PackageId>Microsoft.DotNet.Test.ProjectTemplates.8.0</PackageId>
+        <Authors>Microsoft</Authors>
+        <Description>Test Templates for Microsoft Template Engine</Description>
+        <language>en-US</language>
+        <PackageProjectUrl>https://github.com/dotnet/test-templates</PackageProjectUrl>
+        <CopyrightMicrosoft>© Microsoft Corporation. All rights reserved.</CopyrightMicrosoft>
+        <CopyrightNetFoundation>© .NET Foundation and Contributors</CopyrightNetFoundation>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageType>Template</PackageType>
```

Consider enabling localization
```suggestion
        <PackageType>Template</PackageType>
        <UsingToolTemplateLocalizer>true</UsingToolTemplateLocalizer>
```